include/grpc/grpc_security.h

```diff
@@ -21,6 +21,8 @@
 
 #include <grpc/support/port_platform.h>
 
+#include <stdbool.h>
```

Comment:
Note: This header was added in C99, and historically the C-core headers have been C89, which precludes use of `bool`.  But since we're already in the process of moving from C to C++ for these headers, I think this is fine.

Reply:
Ack SG. If it ends up causing problems and needs to be removed, please let me know and I can refactor to use `int` instead of `bool`.